packages/create-webstone-app/src/tasks/2-setup-project/plugin.ts

```diff
@@ -0,0 +1,89 @@
+//@ts-ignore this package doesn't provdide a declaration file
+import { create } from "create-svelte";
+import { Ctx, getAppName } from "../../helpers";
+import path, { dirname } from "path";
+import fs from "fs-extra";
+import { fileURLToPath } from "url";
+import { ListrTask } from "listr2";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export const copyTemplate = (ctx: Ctx) => {
+  const appDir = ctx.appDir;
+  fs.copySync(
+    path.join(__dirname, "..", "template", "plugin", "structure"),
+    appDir
+  );
+};
+
+export const renameChildPackage = (ctx: Ctx) => {
+  const cliPackageJson = fs.readJSONSync(
+    path.join(ctx.appDir, "packages", "cli", "package.json")
+  );
+  cliPackageJson.name = `webstone-plugin-cli-${getAppName(ctx.appDir)}`;
+  fs.writeJSONSync(
+    path.join(ctx.appDir, "packages", "cli", "package.json"),
+    cliPackageJson,
+    {
+      spaces: "\t",
+    }
+  );
+};
+
+export const renameMainPackage = (ctx: Ctx) => {
+  const packageJson = fs.readJSONSync(path.join(ctx.appDir, "package.json"));
+  packageJson.name = `webstone-plugin-${getAppName(ctx.appDir)}`;
+  fs.writeJSONSync(path.join(ctx.appDir, "package.json"), packageJson, {
+    spaces: "\t",
+  });
+};
+
+export const renamePackages = async (ctx: Ctx) => {
+  renameChildPackage(ctx);
+  renameMainPackage(ctx);
+};
+
+export const createCLINamespace = (ctx: Ctx) => {
```

Comment:
We already turned from space-people to tab-people. Looking at this function name, I feel our next progression is to turn from CamelCase to kebab_case so we don't have to wonder if it should be `createCLINamespace` or `createCliNamespace` :sweat_smile: